app/javascript/modules/listingDetails/ListingDetailsUnitAccordions.tsx

```diff
@@ -47,8 +47,8 @@ const getTableData = (units: RailsListingUnits[]) =>
     area: {
       content: (
         <>
-          <span className="font-semibold">{unit.Unit_Square_Footage}</span>
-          {t("listings.features.sqft")}
+          <span className="font-semibold">{unit.Unit_Square_Footage}</span>{" "}
+          {t("listings.features.squareFeet")}
```

Comment:
this reads out better, but i agree can be long. i'm not sure how the abbreviations would work in the other languages, but according to chrome 'sq ft' reads out as expected -> "two hundred fifty five square feet". i can show you how i'm testing.